src/sonic-config-engine/minigraph.py

```diff
@@ -2040,6 +2040,9 @@ def parse_xml(filename, platform=None, port_config_file=None, asic_name=None, hw
     results['ACL_TABLE'] = filter_acl_table_bindings(acls, neighbors, pcs, pc_members, sub_role, current_device['type'] if current_device else None, is_storage_device, vlan_members)
     results['FEATURE'] = {
         'telemetry': {
+            'state': 'disabled'
+        },
+        'gnmi': {
```

Comment:
>gnmi [](http://example.com/codeflow?start=9&length=4)

We will not modify minigraph.py to enable new features anymore. @wen587 to review.

Reply:
I have removed related changes.